application/src/main/java/org/thingsboard/server/controller/AdminController.java

```diff
@@ -145,6 +149,16 @@ public void sendTestSms(@RequestBody TestSmsRequest testSmsRequest) throws Thing
         }
     }
 
+    @PreAuthorize("hasAuthority('SYS_ADMIN')")
+    @RequestMapping(value = "/cache/attributes/invalidate", method = RequestMethod.DELETE)
+    public void invalidateAttributesCache() throws ThingsboardException {
+        try {
+            clusterService.invalidateAttributesCache();
```

Comment:
This will invalidate cache on a random server in a cluster mode. Not on all servers.